Update INSTALL.md (#6784)
alacritty/alacritty: INSTALL.md

#### Fish

> Make sure you are in fish shell or run
> 
> ```
> fish
> ```
> 
> Once you are in fish shell, to install the completions for fish, run

Reviewer: Maybe something like that? At least it should give you an idea how to adjust it, but the suggestion should be oneline imho.

```suggestion
To install the completions for fish, from inside the fish shell, run
```
